src/box/lua/config/instance_config.lua

```diff
@@ -2351,6 +2351,12 @@ return schema.new('instance_config', schema.record({
         }, {
             default = 'new',
         }),
+        box_error_serialize_verbose = schema.enum({
+            'old',
+            'new',
+        }, {
+            default = 'new',
+        }),
```

Comment:
This change should go after or together with introducing the option in the `compat` module, because otherwise an attempt to set it on this revision leads to an error (and it likely will be catched by `config-luatest/compat_test.lua`. Every commit to `master` should be working and pass tests.

The default must be `old`, see details in the [comment](https://github.com/tarantool/tarantool/pull/9716#discussion_r1500740119) above.

Reply:
Fixed.